lib/src/auth/exception/firestore_exception_code.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:firedart/src/auth/exception/firestore_exception.dart';
+
+/// Represents an exception code of the [FirestoreException].
+enum FirestoreExceptionCode {
+  /// Indicates whether the operation was cancelled.
```

Comment:
```suggestion
  /// An exception code indicating that the operation was cancelled.
```
Please apply to all error codes.